policy/runtime/runtime.go

```diff
@@ -324,6 +334,12 @@ func (t *Template) newEvaluator(mdl *model.Evaluator,
 			decisions: decs,
 		}
 	}
+	// Cost overflows or is greater than the limit.
+	if exprCostLimit >= 0 && (cost < 0 || cost > int64(exprCostLimit)) {
```

Comment:
Could you remind me why the max cost would be negative? Overflow?

Reply:
Yes, overflow.

Reply:
Let's cap the value to the max `int64` value instead of permitting the overflow. I think it will make more intuitive sense to someone trying to figure out why the expression was rejected.

Reply:
Done.